src/utils.ts

```diff
@@ -517,28 +517,30 @@ export default function utils(currentSettings: Settings) {
 			},
 
 			// Get a command's spec
-			getCommandSpec(directory: string): CommandSpec {
+			async getCommandSpec(directory: string): Promise<CommandSpec> {
 				// Error if directory does not exist
 				if (!fs.existsSync(directory)) {
 					throw new Error(`Directory does not exist: ${directory}`);
 				}
 
 				// List the files in this directory
 				const commandFiles: string[] = utils(settings).files.getFiles(directory);
+				const specFiles = commandFiles.filter((path) => path.match(/\.spec.[cm]?js$/));
```

Comment:
Variable 'commandFiles' does not get used beyond this one use.  Instead, you could apppend the `.filter()` directly onto the `.getFiles(directory)` above, and eliminate entirely the `commandFiles` variable.

By the time you format the result, it may well spread the source out over 3 source lines, so it's not less compact, and by having `commandFiles`, it might make what's being done more readable.

Reply:
That's true! I'll make the change.